theHarvester/lib/version.py

```diff
@@ -1,5 +1,7 @@
 # coding=utf-8
 
+VERSION = "4.4.3"
```

Comment:
This was needed for setuptools to discover the dynamic [`version`](https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#dynamic-metadata) (it only supports a static constant, not a callable function)